### PR TITLE
Update geological-sites.ttl

### PR DESCRIPTION
**Pull request by KellyVance**
Addition of notations

### CODE REVIEW

File: vocabularies/geological-sites.ttl [hunk 1]
```diff
@@ -19,6 +19,7 @@
         gsqst:cutting,
         gsqst:field-site,
         gsqst:mine,
+        gsqst:mineral-occurrence
```
Comment (LukeHauck) [inline vocabularies/geological-sites.ttl]: needs a comma at end.

Comment (KellyVance) [reply vocabularies/geological-sites.ttl]: updated.

File: vocabularies/geological-sites.ttl [hunk 2]
```diff
@@ -114,6 +126,7 @@ gsqst:borehole a skos:Concept ;
         "Well"@en ;
     skos:definition "A Borehole is an intended or actual drilled hole in the ground designed to exchange (or facilitate the exchange of) fluids between a subsurface reservoir and the surface (or another reservoir), or to enable the detection and measurement of rock properties.  Boreholes are the sites of extraction or rock samples, as core or cuttings, and fluid samples. A Borehole has a single Origin and all Wellbore(s) that extend from this Well Origin are parts of the Borehole. Sources: PPDM What Is A Well, GSQ."@en ;
```
Comment (GSQ-AI) [inline vocabularies/geological-sites.ttl]: Does this description bend between status and site? 
As in, a borehole is always a hole bored into the Earth, whereas Intended or Actual are states in which the site finds itself. 
I propose we omit "an intended or actual", to reflect that we're describing the concept of what a borehole is. @KellyVance @DavidCrosswellGSQ @dxwell

Comment (DavidCrosswellGSQ) [reply vocabularies/geological-sites.ttl]: Boy, this definition sure has had some iterations :)

I tend to agree with the omit - is this like Schrodinger's cat?

Comment (KellyVance) [reply vocabularies/geological-sites.ttl]: I suggest omitting "intended or actual", to make the concept definition more robust.
But we should add a scopeNote to explicitly call out that as a regulatory agency for our purposes the borehole exists once it has been proposed, prior to it being physically drilled.

**Pull request merged**